rock-paper-scissors.js

```diff
@@ -0,0 +1,44 @@
+// 7. In this kata, your task is to implement an extended version of the famous rock-paper-scissors game.
+
+function willRockWin(player) {
```

Comment:
I would call it an `opponent` instead of a `player` to emphasize the fact that this is an opponent of the rock.

Reply:
Ok, I understand.